TruckLoadingApp.Client/src/css/Landing.css

```
@@ -0,0 +1,386 @@
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  
+  /*content container */
+  .hero-content {
+    z-index: 10;
+    width: 100%;
+    max-width: 7xl;
```

Biome (1.9.4)

[error] Unexpected unknown unit: xl

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)